packages/babel-helper-module-transforms/src/rewrite-live-references.ts

```diff
@@ -44,6 +44,25 @@ interface RewriteBindingInitVisitorState {
   scope: Scope;
 }
 
+function isInType(path) {
+  do {
+    switch (path.parent.type) {
+      case "TSTypeAnnotation":
+      case "TSTypeAliasDeclaration":
+      case "TSTypeReference":
+      case "TypeAnnotation":
+      case "TypeAlias":
```

Comment:
we don't have an alias to cover all type usage for flow/ts? (or is this it anyway)

Reply:
It's enough to cover the nodes where the AST stops being an expression and starts being a type. The problem with a `TypeScript`/`Flow` catch-all is that both TS and Flow introduce some expression nodes (for example, in enums).